mne/decoding/tests/test_base.py

```diff
@@ -12,6 +13,48 @@
 from mne.decoding import Scaler
 
 
+def _make_data(n_samples=1000, n_features=5, n_targets=3):
+    """Generate some testing data.
+
+    Parameters
+    ----------
+    n_samples : int
+        The number of samples.
+    n_features : int
+        The number of features.
+    n_targets : int
+        The number of targets.
+
+    Returns
+    -------
+    X : ndarray, shape (n_samples, n_features)
+        The measured data.
+    Y : ndarray, shape (n_samples, n_targets)
+        The latent variables generating the data.
+    A : ndarray, shape (n_features, n_targets)
+        The forward model, mapping the latent variables (=Y) to the measured
+        data (=X).
+    """
+
+    # Define Y latent factors
+    np.random.seed(0)
+    cov_Y = np.eye(n_targets) * 10 + np.random.rand(n_targets, n_targets)
+    mean_Y = np.random.rand(n_targets)
+    Y = np.random.multivariate_normal(mean_Y, cov_Y, size=n_samples)
+
+    # Put a scale and offset to Y
+    Y += np.random.rand(n_targets)  # Put an offset
```

Comment:
this line is actually not necessary. You already specify a non-zero mean on line 42.